packages/composite-checkout-wpcom/src/components/wp-order-review-line-items.js

```diff
@@ -32,12 +31,12 @@ const OrderReviewSectionArea = styled.div`
 	margin-bottom: 16px;
 `;
 
-function WPLineItem( { item, className, hasDeleteButtons, removeProduct } ) {
+function WPLineItem( { item, className, hasDeleteButtons, removeItem } ) {
 	const translate = useTranslate();
-	const hasDomainsInCart = useHasDomainsInCart();
 	const itemSpanId = `checkout-line-item-${ item.id }`;
 	const deleteButtonId = `checkout-delete-button-${ item.id }`;
 	const [ isModalVisible, setIsModalVisible ] = useState( false );
+	const hasDomainsInCart = useHasDomainsInCart();
```

Comment:
Super minor, but is there any reason for moving this down, or was it just an artifact of refactoring? (In a big PR it's nice to avoid any unnecessary changes to make it easier to review.)

Reply:
It's an artifact- thanks for the heads up.